### DISCUSSION TITLE
Camoufox.com Launched 🎉

### DISCUSSION THREAD

**Discussion by daijro** [daijro/camoufox#39] [Announcements]
Hello!

I'd like to announce the launch of https://camoufox.com/ as a new source for Camoufox's documentation.

### Better Docs

I realize the currently documentation (giant GitHub READMEs) is extremely painful to navigate and can be difficult for new people to read through. The website uses [Retype](https://retype.com/) for (hopefully) better styling and making it easier to jump around and search the docs.

### WebGL Analytics

I will also be using this to help gather WebGL info (with the user's consent of course) to eventually integrate WebGL rotation/fingerprint generation similar to BrowserForge (#32). To do this, I'll need to assemble a database of WebGL info for different GPU models. If anyone would like to help me integrate this, feel free to stop by the website and allow the analytics popup!

---

### What's planned?

As I move forward, I plan on adding anti-bot/fingerprinting debugging sites to help others making similar bot/fingerprint detection projects similar to the [Camoufox WebGL test](https://camoufox.com/tests/webgl).

**Comment by cheryiit** [comment]
Hi Can i look the docs can you create branch for that?
Because i am developing with claude code and i need a docs like a llm.txt.

So can you share us pls?
